_codux/ui-kit/buttons/buttons.board.tsx

```diff
@@ -30,10 +30,16 @@ export default createBoard({
                 </Kit.Item>
                 <Kit.Item>
                     <Variant name="Secondary Button">
-                        <button className="button secondary">Add to Cart</button>
+                        <button className="button">Add to Cart</button>
                     </Variant>
                     <Kit.Description>Secondary Button</Kit.Description>
                 </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Secondary Muted Button">
+                        <button className="button mutedSecondaryButton">Checkout</button>
```

Comment:
btw, maybe write some neutral text in the button: "Button", variant name, "Your label"? Or to do it together with other buttons in separate PR?